<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>saas_guide_hard_test</class>
 <widget class="QWidget" name="saas_guide_hard_test">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0, 0, 0, 0;
border-left:2px solid gray;</string>
   </property>
   <property name="text">
    <string>打印机</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_printer_test">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>80</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>测试</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0, 0, 0, 0;
border-left:2px solid gray;</string>
   </property>
   <property name="text">
    <string>钱箱</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cdrawer_test">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>130</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>测试</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_kexian_type">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>210</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0, 0, 0, 0;
border-left:2px solid gray;</string>
   </property>
   <property name="text">
    <string>客显类型</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_printer_manage">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>80</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_cdrawer">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cdrawer_default">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>130</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>默认</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>210</y>
     <width>61</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>端口</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_kexian_port">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>210</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_display_test">
   <property name="geometry">
    <rect>
     <x>429</x>
     <y>210</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>测试</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>381</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0, 0, 0, 0;
border-left:2px solid gray;</string>
   </property>
   <property name="text">
    <string>读卡器</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_beep">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>270</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>蜂鸣</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_printer">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>/dev/usb/lp0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>/dev/ttyS3,115200</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>481</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:20pt;</string>
   </property>
   <property name="text">
    <string>硬件检测</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>510</width>
     <height>3</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#ff546b;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
